Framework/Geometry/inc/MantidGeometry/Instrument/DetectorInfoItem.h

```diff
@@ -0,0 +1,78 @@
+#ifndef MANTID_GEOMETRY_DETECTORINFOITEM_H_
+#define MANTID_GEOMETRY_DETECTORINFOITEM_H_
+
+#include "MantidGeometry/Instrument/DetectorInfo.h"
+#include "MantidKernel/Quat.h"
+#include "MantidKernel/V3D.h"
+
+#include <utility>
+
+using Mantid::Geometry::DetectorInfo;
+using Mantid::Kernel::Quat;
+using Mantid::Kernel::V3D;
+
+namespace Mantid {
+namespace Geometry {
+
+class MANTID_GEOMETRY_DLL DetectorInfoItem {
+
+public:
+  DetectorInfoItem(const DetectorInfoItem &other) = default;
```

Comment:
Both copyable and moveable. A bit of an advanced question, but do you understand why you need to provide both to clients?